src/main/groovy/org/labkey/gradle/task/CreateModule.groovy

```diff
@@ -0,0 +1,152 @@
+package org.labkey.gradle.task
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.CopySpec
+import org.gradle.api.tasks.TaskAction
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+/**
+ * Creates a new module based on the contents of trunk/server/moduleTemplate. Sources properties from commandline or
+ * user input based on the prompt flag.
+ */
+class CreateModule extends DefaultTask
+{
+    boolean prompt = true
+
+    @TaskAction
+    void createModule() {
+        String moduleName = project.hasProperty('moduleName') ? project.moduleName : null
+        String moduleDestination = project.hasProperty('moduleDestination') ? project.moduleDestination : null
+        boolean hasManagedSchema = project.hasProperty('noManagedSchema')
+        boolean createTestFiles = project.hasProperty('createTestFiles')
+        boolean createApiFiles = project.hasProperty('createApiFiles')
```

Comment:
Instead of using the boolean properties, how about changing to a different kind of property with a value, such as: -Pfiles=test,api,schema.  

Then we extract the boolean values from this property and if there is no such property we know we need to prompt for answers. Then we don't need two tasks.  (I think probably only TeamCity uses the noPrompt task, so it's not a big deal, but if we can make it more friendly to humans to use, they may do so, and then we don't have special logic just for TeamCity that we might forget it needs.)